src/com/carolinarollergirls/scoreboard/core/impl/FieldingImpl.java

```diff
@@ -114,7 +121,12 @@ public void updateBoxTripSymbols() {
         for (ValueWithId v : getAll(Child.BOX_TRIP)) {
             trips.add((BoxTrip) v);
         }
-        Collections.sort(trips);
+        Collections.sort(trips, new Comparator<BoxTrip>() {
+            public int compare(BoxTrip b1, BoxTrip b2) {
+                if (b1 == b2) { return 0; }
```

Comment:
This is redundant

Reply:
It catches the case where b1 == b2 == null.

Reply:
That's already covered by the next two lines

Reply:
No. The next line would return 1 instead of 0 in that case.